pages/about/index.vue

```diff
@@ -2,20 +2,54 @@
   <div class="about-page">
     <breadcrumb :breadcrumb="breadcrumb" :title="pageTitle" />
     <page-hero v-if="heroCopy">
-      <h1>{{ pageTitle }}</h1>
-      <p>
-        {{ heroCopy }}
-      </p>
+      <div class="row">
+        <div class="col">
+          <h1>{{ pageTitle }}</h1>
+          <p>
+            {{ heroCopy }}
+          </p>
+          <button class="about-page-button">
+            <a href="https://commonfund.nih.gov/sparc/" target="_blank">More Info NIH Website</a>
+          </button>
+        </div>
+        <div class="col">
+          <img class="page-hero-img" src="@/static/images/about-hero.png" />
+        </div>
+      </div>
     </page-hero>
     <div class="page-wrap container">
       <div class="subpage">
-        <el-row type="flex" justify="center">
-          <el-col :row="24">
-            <!-- eslint-disable vue/no-v-html -->
-            <!-- marked will sanitize the HTML injected -->
-            <div v-html="parseMarkdown(copy)" />
-          </el-col>
-        </el-row>
+        <div class="row">
+          <div class="col">
+            <div class="about-page-text" v-html="parseMarkdown(overview)" />
+          </div>
+          <div class="col">
+            <!-- <img src="https://via.placeholder.com/222" /> -->
+          </div>
+        </div>
+      </div>
+      <div class="subpage">
+        <div class="row">
+          <div class="col">
+            <!-- <img class="about-page-portal-img" src="https://via.placeholder.com/222" /> -->
+          </div>
+          <div class="subpage-col">
+            <p class="about-page-text" v-html="parseMarkdown(sparcPortal)" />
+            <!-- <img class="about-page-portal-img-mobile" src="https://via.placeholder.com/222" /> -->
+            <button class="about-teams-and-projects-button">
+              <nuxt-link
```

Comment:
This link and copy should come from contentful as well.

Reply:
I'll change the link, but the copy is coming from contentful. The About page has two fields, one called `overview` and the other `sparcPortal`

Reply:
I mean the actual label "View Teams & Projects" of thee button.